config/default/field.field.paragraph.hs_collection.field_title_settings.yml

```diff
@@ -12,7 +12,7 @@ field_name: field_title_settings
 entity_type: paragraph
 bundle: hs_collection
 label: 'Title Settings'
-description: ''
+description: 'See the <a href="https://hsweb.slite.page/p/mc94Sg92YTiY7V/Collection-Guide#23102d46">User Guide</a> for help on how to use this option.'
```

Comment:
Please add a `target="_blank"` to the link to have it open in a new tab.

Reply:
Fixed!